doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1459,3 +1452,12 @@ Here are a few examples:
         Proof.
           now foo.
         Abort.
+
+.. [1]
+   This is similar to the expression “*entry* :math:`\{` sep *entry*
+   :math:`\}`” in standard BNF, or “*entry* :math:`(` sep *entry*
+   :math:`)`\ \*” in the syntax of regular expressions.
```

Comment:
Why do we use math rather than code for bnf?

Reply:
I had missed this comment, and indeed I can change this if you want.

Reply:
I'd rather merge this and address the rest in a separate PR :)